doc/SwaggerApi.xml

```diff
@@ -0,0 +1,165 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: YnvestTubeApi
+host: "127.0.0.1:8000"
+consumes:
+  - "application/json"
+produces:
+  - "application/json"
+paths:
+  /accounts/register:
+    get:
+      summary: "Register new user"
+      operationId: "registerUser"
```

Comment:
This is a little personal, but for me it is better to stick to one convention.

I mean you used `accounts` in paths but then in summary and operationId you have used `user`

Maybe change accounts to users or users to accounts.

I feel like it's semantic mistake

Reply:
changed paths to users

Reply:
and removed operationId. I dscovered it is actually senseless and doesn't change anything in the appearance